chevah_build

```diff
@@ -355,8 +355,10 @@ help_text_test=\
 "Run a quick test for the Python from build."
 command_test() {
     test_file='test_python_binary_dist.py'
+    test_file_helper='get_binaries_deps.sh'
```

Comment:
test_file_helper ? I think that this is a bad name for this variable.... since this is used in a single place, do we need a variable for it?